lib/ziti-tunnel/tunnel_udp.c

```diff
@@ -3,6 +3,14 @@
 #include "tunnel_udp.h"
 #include "ziti_tunnel_priv.h"
 
+#define UDP_TIMEOUT 30
+
+// initiate orderly shutdown
```

Comment:
Will it be ok if this happens while the connection has pending writes (e.g. waiting for router reconnect)?

Reply:
Timer is started/reset when actual payload is written to either direction